### ISSUE TITLE
Fine tuning hyper-parameter

### ISSUE THREAD

**Issue by Mauwt** [CS1103/MNISTplusplus#27]
## **THIS TASK DOES NOT INVOLVE CODE**

The team should decide the following hyper-parameters:
  - [x] Learning rate
  - [x] Number of layers 
  - [x] Number of neurons per layer
  - [x] Number of training epochs
